workitem/workitem_repository.go

```diff
@@ -770,6 +777,13 @@ func (r *GormWorkItemRepository) GetCountsPerIteration(ctx context.Context, spac
 		spaceID.String())
 	db = r.db.Raw(queryIterationWithChildren)
 	db.Scan(&itrChildren)
+	if db.Error != nil {
+		log.Error(ctx, map[string]interface{}{
+			"spaceID": spaceID.String(),
```

Comment:
done in https://github.com/almighty/almighty-core/pull/1151/commits/f11d893ea23493f2a73f28b4fed578a77331e80c